ext/tags.go

```diff
@@ -23,25 +23,54 @@ var (
 
 	// SamplingPriority determines the priority of sampling this Span.
 	SamplingPriority = uint16Tag("sampling.priority")
+
+	// SpanKind hints at relationship between spans, e.g. client/server
+	SpanKind = spanKindTag("span.kind")
+
+	// SpanKindRPCClient marks a span representing the client-side of an RPC
+	// or other remote call
+	SpanKindRPCClient = SpanKindEnum("c")
+
+	// SpanKindRPCServer marks a span representing the server-side of an RPC
+	// or other remote call
+	SpanKindRPCServer = SpanKindEnum("s")
 )
 
+// ---
+
+// SpanKindEnum represents common span types
+type SpanKindEnum string
```

Comment:
I've added new values using the same pattern used by the other ones. That pattern has been agreed upon back in Dec https://github.com/opentracing/opentracing-go/commit/3e2f0d7c16a7aa88576ef114a638cbaf3471f8c9#diff-0c56d0f1cdc8e3b8ba3be800367de5ee

If you think it's worth changing, e.g. from performance considerations, I think it should be a different PR.